### PR TITLE
Add v1 API

### PR DESCRIPTION
**Pull request by samkim**
* Add v1 API
* Add v1 example (uses TLS)
* Add integration test and workflow (runs against spicedb test server over plain text) 
* Remove v0 and v1alpha1 examples

### CODE REVIEW

**Comment by alexisvincent** [comment]
🎉🎉🎉

File: examples/v1/App.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        ManagedChannel channel = ManagedChannelBuilder
+                .forTarget(target)
+                .useTransportSecurity()
+                .build();
+        try {
+            App client = new App(channel);
+            client.writeSchema();
+            client.readSchema();
+            String tokenVal = client.writeRelationship();
+            client.check(
```
Comment (josephschorr) [inline examples/v1/App.java:59]: Newline between the various operations

Comment (josephschorr) [reply examples/v1/App.java:59]: Print out the check result?

**Review by josephschorr — COMMENTED** [review]
Example code looks good

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**